hwdb.d/60-keyboard.hwdb

```diff
@@ -1348,6 +1348,40 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnOQO*Inc.*:pnOQO*Model*2*:pvr*
  KEYBOARD_KEY_f2=volumedown
  KEYBOARD_KEY_f3=volumeup
 
+###########################################################
+# Ortek
+###########################################################
+
+# Adesso AKB-805MAC
+# The key code identifiers used below exactly match the text
+# labels on the keys/buttons (or standard icons on the buttons
+# that have no text labels), except as noted.
+evdev:input:b0003v05A4p9735*
+ KEYBOARD_KEY_c0015=back
+ KEYBOARD_KEY_c0012=forward
+ KEYBOARD_KEY_c000c=stop
+ KEYBOARD_KEY_c0018=refresh
+ KEYBOARD_KEY_c00b9=search
+ KEYBOARD_KEY_c0006=bookmarks    # Button labeled "Favorites"
+ KEYBOARD_KEY_c00a8=homepage     # Button labeled "Web/Home"
+ KEYBOARD_KEY_c0010=mute
+ KEYBOARD_KEY_c0011=volumedown
+ KEYBOARD_KEY_c0014=volumeup
+ KEYBOARD_KEY_c000e=close
+ KEYBOARD_KEY_c00a7=print
+ KEYBOARD_KEY_c0013=documents    # Button labeled "Stickies"
+ KEYBOARD_KEY_c000d=find         # Button labeled "Sherlock2"
```

Comment:
That one killed me :)

Reply:
Glad an otherwise mundane PR could provide some enjoyment. Thanks so much for landing this!